checkpoints.go

```diff
@@ -118,6 +122,7 @@ func capture(
 		ExpressionAttributeValues: attrVals,
 	}); err != nil {
 		if awsErr, ok := err.(awserr.Error); ok && awsErr.Code() == "ConditionalCheckFailedException" {
+			logger.Log("failed to capture shard '%s'", shardID)
```

Comment:
Could be useful to print the error message too

```suggestion
			logger.Log("failed to capture shard '%s': %s", shardID, awsErr)
```